ROSCO_testing/ROSCO_testing.py

```diff
@@ -296,6 +304,8 @@ def ROSCO_Test_heavy(self, more_case_inputs={}, U=[]):
         iec.dlc_inputs['Yaw'] = [[], []]
         iec.transient_dir_change = '-'  # '+','-','both': sign for transient events in EDC, EWS
         iec.transient_shear_orientation = 'v'  # 'v','h','both': vertical or horizontal shear for EWS
+        iec.uniqueSeeds = True
```

Comment:
This seems like it needs the unique seeds related pull request to go through in WEIS - yes?

Reply:
Having this here without the related pull request in WEIS will not break anything, but it won't generate unique wind seeds without that PR.